### ISSUE TITLE
Better EMACS support

### ISSUE THREAD

**Issue by nrdxp** [divnix/digga#173]
I'm not an EMACS users, but after some discussion with others, it seems to be a common pain point to set up in NixOS in general, and perhaps DevOS could offer those users guidance with boilerplate and docs. I'll ping some of the others I've worked with to brainstorm here. I'll be happy to delegate a PR to a more experienced emacs user, once an implementation is ready.

From what I've seen so far, home-manager helps a lot, and perhaps a pure hm implementation would allows users to take their emacs to any system with nix installed.

/cc @codygman @unrooted

**Comment by codygman** [comment]
In the past I used a pure home manager emacs config and it made things pretty simple. I never really tried the nixpkgs one alone though, so i guess I can't say?

**Comment by Unrooted** [comment]
Basically seems like pure home-manager config is the easiest to set up and get it running. I've tried doom-emacs, worked _almost_ fine with home manager, except for some things, though this may be mine fault, since I may have did something in a wrong way by mistake. But when I've switched to flakes, it kinda messed up and seems like it can be more complex, even though the doom-emacs is a flake itself. But as I said, maybe I have done something in a wrong way in here as well. I'm still learning, so keep that in mind, that I may have been doing something wrong all the time, but anyway, it would be cool to make things easier to set up.
